scripts/commands.py

```diff
@@ -103,17 +103,3 @@ def _print(message):
             _print('Empty list of blackout_dates in Diablo config file.')
 
         _print('Have a nice day!')
-
-
-@application.cli.command('delete_kaltura_event_by_id')
-@click.argument('event_id')
-def delete_kaltura_event(event_id):
-    """Delete Kaltura event."""
-    with application.app_context():
-        from diablo.externals.kaltura import Kaltura
-
-        Kaltura().delete_event(kaltura_schedule_id=event_id)
-        print(f"""
-            Kaltura event {event_id} deleted.
-            Have a nice day.
-        """)
```

Comment:
Superfluous. You can delete an event most easily with https://developer.kaltura.com/api-docs/service/scheduleEvent/action/delete